pkg/healthcheck/healthcheck.go

```diff
@@ -132,11 +132,18 @@ const (
 	// is correct. These checks are no ops if linkerd is not in HA mode
 	LinkerdHAChecks CategoryID = "linkerd-ha-checks"
 
+	// LinkerdCniPluginChecks adds checks to validate that the CNI
+	/// plugin is installed and ready
+	LinkerdCniPluginChecks CategoryID = "linkerd-cni-plugin"
```

Comment:
Can we follow the Go code review convention on [initialism](https://github.com/golang/go/wiki/CodeReviewComments#initialisms) here and elsewhere? So `LinkerdCNIPluginChecks`, instead of `LinkerdCniPluginChecks`? Thanks.